README.md

```diff
@@ -41,6 +41,25 @@ For an even easier usage, just `#import <KVOController/NSObject+FBKVOController.
 [self.KVOController observe:clock keyPath:@"date" options:NSKeyValueObservingOptionInitial|NSKeyValueObservingOptionNew action:@selector(updateClockWithDateChange:)];
 ```
 
+## Swift
+
+KVOController works fine with Swift but the properties that you need to observer must be marked as `dynamic` check the following example:
+```Swift
+class TasksListViewModel: NSObject, TasksListViewModelProtocol{
+
+    dynamic var tasksList: [TaskList] = []
+}
+
+/// In ViewController.swift
+
+import KVOController
+
+self.KVOController.observe(self.viewModel as? AnyObject, keyPath: "tasksList", options: [.New, .Initial]) { (viewController, viewModel, change) in
```

Comment:
Updated to Swift 3. The 'as? AnyObject' is added because the view model property was defined in my project like this:
`var viewModel: TasksListViewModelProtocol?`